src/views/articles.hbs

```diff
@@ -0,0 +1,17 @@
+<div id="search">
+  <form class="search__form" action="/articles" method="POST">
+     <input class="search__input" type="text" name="search" placeholder="Search by
```

Comment:
in the placeholder why is there so many spaces ?

Reply:
@Karyum deleted unnecessary space